pkgs/shells/zsh/default.nix

```diff
@@ -1,23 +1,19 @@
 { lib, stdenv, fetchurl, ncurses, pcre, buildPackages }:
 
-let
-  version = "5.8";
-
-  documentation = fetchurl {
-    url = "mirror://sourceforge/zsh/zsh-${version}-doc.tar.xz";
-    sha256 = "1i6wdzq6rfjx5yjrpzan1jf50hk2pfzy5qib9mb7cnnbjfar6klv";
-  };
-in
-
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "zsh";
-  inherit version;
+  version = "5.8";
 
   src = fetchurl {
     url = "mirror://sourceforge/zsh/zsh-${version}.tar.xz";
     sha256 = "09yyaadq738zlrnlh1hd3ycj1mv3q5hh4xl1ank70mjnqm6bbi6w";
   };
 
+  documentation = fetchurl {
```

Comment:
If you're not using the `$documentation` bash variable, you shouldn't create it.
Please revert.

Reply:
it is used in line 35

Reply:
Line 35 uses the Nix variable, so it doesn't have to be added to bash via `mkDerivation { documentation = x; }`. Please put it back in its `let` binding.